paig-server/backend/paig/api/shield/scanners/PIIScanner.py

```diff
@@ -123,8 +123,14 @@ def __init__(self, name, request_types, enforce_access_control, model_path, mode
         """
         Initialize the required models and variables for the scanner
         """
-        super().__init__(name, request_types, enforce_access_control, model_path, model_score_threshold, entity_type,
-                         enable, **kwargs)
+        super().__init__(name=name,
+                         request_types=request_types,
+                         enforce_access_control=enforce_access_control,
+                         model_path=model_path,
+                         model_score_threshold=model_score_threshold,
+                         entity_type=entity_type,
+                         enable=enable,
+                         **kwargs)
```

Comment:
In the base class constructor we are taking only **kwargs, see if we can take all paramteres in **kwargs only in this constructor as well

Reply:
updated